chef/cookbooks/suse-manager-client/recipes/default.rb

```diff
@@ -36,11 +38,10 @@
   EOH
```

Comment:
So you think just installing `ssl-cert.rpm` and calling `update-ca-certificates`? Or is some other step needed?